closure/goog/editor/field.js

```diff
@@ -2799,4 +2801,4 @@ goog.editor.Field.prototype.getIframeAttributes = function() {
   }
 
   return {'frameBorder': 0, 'style': iframeStyle};
-};
+};
```

Comment:
@gmalartre you are looking at the wrong line
 Before you squashed you had one commit that added two spaces and one commit that removed two spaces.
In this change you simply removed the new line in the end of the file.